source/data_analysis_spring/data_analysis_spring_2023.Rmd

```diff
@@ -1083,6 +941,275 @@ new_data |>
+```{r}
+emmeans::emmeans(model5, pairwise ~ cumulative_days,
+                 type = "response",
+                 at = list(cumulative_days = c(0.25,
+                                               1.25,
+                                               2.25,
+                                               3.25,
+                                               4.25,
+                                               5.25,
+                                               6.25,
+                                               7.25,
+                                               8.25,
+                                               9.25)))
+```
+
+```{r, fig.cap = "Proportie soorten gevonden over tijd voor wilde bijen."}
+pp <- marginaleffects::predictions(
+  model5,
+  by = c("cumulative_days", "method_combi"),
+  type = "link",
+  vcov = TRUE,
+  re.form = NA
+)
+
+pp <- pp |>
+  mutate(
+    prop = plogis(estimate),
+    lower = plogis(conf.low),
+    upper = plogis(conf.high)
+  )
+
+pp |>
+  ggplot() +
+  geom_line(
+    aes(x = cumulative_days, y = prop, colour = method_combi)
+  ) +
+  geom_ribbon(
+    aes(
+      x = cumulative_days,
+      ymin = lower,
+      ymax = upper,
+      fill = method_combi),
+    alpha = 0.2
+  ) +
+  labs(y = "Proportion of species found",
+       x = "Cumulative days")
+
+```
+
+## Zweefvliegen
+
+```{r}
+# Add unique identifier for each sampling site and method combination
+# and find the number of species for the highest number of days
+syrphidae_cumulative <- syrphidae_cumulative |>
+  mutate(
+    sampling_site_method = paste0(
+      sampling_site_cd, "_", spring_code)) |>
+  group_by(sampling_site_method) |>
+  mutate(n_species_final = max(cumulative_n_species)) |>
+  ungroup() |>
+  mutate(prop = cumulative_n_species / n_species_final)
+```
+
+```{r}
+model6 <- glmmTMB(prop ~ cumulative_days * method_combi +
+                    (1 | sampling_site_method),
+                  family = binomial,
+                  weights = n_species_final,
+                  data = syrphidae_cumulative)
+```
+
+```{r}
+# check model assumptions
+performance::check_model(model6)
+```
+
+Verwacht percentage van de "totale" gemeenschap dat gevonden wordt voor het aantal dagen dat pan traps geplaatst worden:
+```{r}
+emmeans::emmeans(model6, pairwise ~ cumulative_days,
+                 type = "response",
+                 at = list(cumulative_days = c(0.25,
+                                               1.25,
+                                               2.25,
+                                               3.25,
+                                               4.25,
+                                               5.25,
+                                               6.25,
+                                               7.25,
+                                               8.25,
+                                               9.25)))
+```
+
+```{r, fig.cap = "Proportie soorten gevonden over tijd voor zweefvliegen."}
```

Comment:
Best een chunk-label toevoegen bv time-series-dragonflies. Op die manier krijgen de figuren in de map met figuren ook een logische naam. Ook hierboven (en mogelijk elders) best aanpassen.